src/Tickets/Commerce/Reports/Data/Order_Summary.php

```diff
@@ -0,0 +1,265 @@
+<?php
+namespace TEC\Tickets\Commerce\Reports\Data;
+
+use TEC\Tickets\Commerce\Status\Completed;
+use TEC\Tickets\Commerce\Status\Status_Handler;
+use TEC\Tickets\Commerce\Ticket;
+use TEC\Tickets\Commerce\Utils\Value;
+use Tribe__Tickets__Ticket_Object as Ticket_Object;
+use Tribe__Tickets__Tickets;
+
+/**
+ * Class Order_Summary.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Reports\Data
+ */
+class Order_Summary {
+	/**
+	 * @var int The post ID.
+	 */
+	protected int $post_id;
+
+	/**
+	 * @var Ticket_Object[] The tickets.
+	 */
+	protected array $tickets = [];
+
+	/**
+	 * @var array|array[] The tickets by type.
+	 */
+	protected array $tickets_by_type = [];
+
+	/**
+	 * @var array|array[] The event sales by status.
```

Comment:
```suggestion
	 * @var array<string,array{qty_sold: int, total_sales_amount: int, total_sales_price: int}> The event sales by status.
```
Here and elsewhere document the keys when there is a reasonable expectations, from the method body, about what they will be.